secondbase.gemspec

```diff
@@ -16,12 +16,8 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.rdoc_options  = ['--charset=UTF-8']
   s.license       = 'MIT'
-  s.add_runtime_dependency     'rails', '~> 4.0'
+  s.add_runtime_dependency     'rails'
```

Comment:
Should we change this to `>= 4.0` just in case someone with Rails 2.3 rolls along here :)